.pre-commit-config.yaml

```diff
@@ -3,45 +3,19 @@
 # SPDX-License-Identifier: Unlicense
 
 repos:
-  - repo: https://github.com/python/black
-    rev: 24.2.0
-    hooks:
-      - id: black
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
-    hooks:
-      - id: isort
-        args: ["--profile", "black", "--filter-files"]
-  - repo: https://github.com/fsfe/reuse-tool
-    rev: v1.1.2
-    hooks:
-      - id: reuse
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.4.0
     hooks:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
-  - repo: https://github.com/pycqa/pylint
-    rev: v2.17.4
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.1.3
```

Comment:
This appears to automatically redirect to `https://github.com/astral-sh/ruff-pre-commit` I assume Github will keep that redirect active, but it's kind of an undocumented oddity for anyone that runs across it looking for the current name. I would recommend changing here to the current URL instead of relying on the redirect. 

I noticed they are also up to release `0.3.2`, is there any reason why we would need to use 0.1.3 instead of the newer / current release?

Reply:
Based on comments, I grabbed what MicroPython was using. Totally forgot to mention that.